src/kaspar_lilja_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+# This program makes you guess the sum of two dices.
+# You have 3 tries to guess the right answer.
+
+from random import randint
+
+
+def players_guess():
+    c = 0
+    while c < 1:
+        c = int(input('Your guess: '))
+    return c
+
+
+def dices_sum():
+    return randint(1, 6) + randint(1, 6)
+
+
+def if_correct_answer(f, g):
```

Comment:
avoid if, use is instead. Though note that `is_equal()` will be even better, as that is all the function does.